docker/sink_service/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG GATEWAY_BUILD_SHA1=unset
 
 RUN adduser --disabled-password wirepas
 
-RUN apk add --no-cache gcc make cmake musl-dev dpkg elogind-dev linux-headers
+RUN apk add --no-cache gcc git make cmake musl-dev dpkg elogind-dev linux-headers
```

Comment:
Git required now here in the builder due to the change related to submodule usage?